test/test-functions

```diff
@@ -1442,8 +1442,10 @@ create_empty_image() {
         fi
     fi
 
-    if [ "$IMAGE_NAME" = "repart" ]; then
+    if [[ "$IMAGE_NAME" == "repart" ]]; then
         root_size=$((root_size+=1000))
+    elif [[ "$IMAGE_NAME" == "cryptsetup" ]]; then
```

Comment:
Rather than conditionalize with the image name, may be better to increase size when e.g. `$IMAGE_ADDITIONAL_ROOT_SIZE` is specified, then set the variable in repart test and cryptsetup test?